### PR TITLE
#73 Add event status on admin events list page

### PR DESCRIPTION
**Pull request by denys-medynskyi**

### CODE REVIEW

File: app/views/admin/visit_request/visit_requests/index.slim [hunk 1]
```diff
@@ -6,10 +6,12 @@ table.table
...
       th
+      th = t 'words.status'
 
   tbody
     - visit_requests.each do |visit_request|
       tr
         th = admin_member_link(visit_request.user)
         th = admin_approve_visit_request_link(event, visit_request)
         th = admin_cancel_visit_request_link(event, visit_request)
+        th = visit_request.status.to_s
```
Comment (sveredyuk) [inline app/views/admin/visit_request/visit_requests/index.slim]: bootstrap4 has cute colored labels. what about to use it ? (ofc by helpers)
```html
<td class="bg-info">...</td>
```

**Review by sveredyuk — APPROVED** [review]

File: app/views/admin/visit_request/visit_requests/index.slim (hunk 1 above)
Comment (denys-medynskyi) [reply app/views/admin/visit_request/visit_requests/index.slim]: @sveredyuk do you mean canceled should be red, approved green etc. ?

Comment (sveredyuk) [reply app/views/admin/visit_request/visit_requests/index.slim]: yes.. Usability...

File: app/helpers/admin/visit_requests_helper.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  module VisitRequestsHelper
+
+    STATUS_CLASS = {
+        ::VisitRequest::PENDING => 'bg-info',
+        ::VisitRequest::APPROVED => 'bg-success',
+        ::VisitRequest::CANCELED => 'bg-danger'
+    }
+
+    def status_cell(status)
+      content_tag :td, class: STATUS_CLASS[status.to_sym] do
```
Comment (denys-medynskyi) [inline app/helpers/admin/visit_requests_helper.rb:11]: @sveredyuk I am not using helper modules to extract view logic, so show me the path if this is wrong way

**Pull request merged**